doc/ExpertOptions.md

```diff
@@ -1132,23 +1174,9 @@ In order to override the defaults for specific tasks, you can add the `perTaskDx
 }
 ```
 
-will override the default timeout for tasks `Add` and `Inc`. It will also provide `UPLOAD` instead of `VIEW` project access to `Inc`.
+will override the default task timeout and `dx_timeout` in the runtime section of tasks `Add` and `Inc`. It will also provide `UPLOAD` instead of `VIEW` project access to `Inc`.
 
-You are also able to specify metadata for tasks in the `defaultTaskDxAttributes` and `perTaskDxAttributes` sections, including adding citation or license information.
-
-The full set of attributes that may be specified are:
-
-* title
-* summary
-* description
-* developerNotes
-* version
-* categories
-* types
-* tags
-* properties
-* details
-* openSource
+You are also able to specify metadata for tasks in the `defaultTaskDxAttributes` and `perTaskDxAttributes` sections, including adding citation or license information. The full set of attributes that may be specified are those allowed in the [dxapp.json](https://documentation.dnanexus.com/developer/apps/app-metadata) file.
```

Comment:
@emiloslavsky I added additional comments here to address that here the attribute names should be the same as in dxapp.json, and another comment in line 568 to note the different representation of attrs:
https://github.com/dnanexus/dxCompiler/blob/14d33109765b365805646adf86941660bb563473/doc/ExpertOptions.md?plain=1#L564-L592

Reply:
https://documentation.dnanexus.com/developer/apps/app-metadata has inputSpec, resources, ignoreReuse.  Are all these supported?  If not may be say something like 
```suggestion
You are also able to specify metadata for tasks in the `defaultTaskDxAttributes` and `perTaskDxAttributes` sections, including adding citation or license information. The full set of recognized attributes is listed in [task meta](#meta-section) with snake_case fields converted to camelCase to agree with [dxapp.json](https://documentation.dnanexus.com/developer/apps/app-metadata) syntax (e.g.  `developer_notes` and `open_source` meta attributes are specified as `developerNotes` and `openSource` in `extras.json`).
```